src/font/Metrics.zig

```diff
@@ -144,11 +155,19 @@ pub const FaceMetrics = struct {
         return 0.75 * self.capHeight();
     }
 
+    /// Convenience function for getting the j-height. If we couldn't
+    /// measure this, we estimate it as ascent - descent.
+    pub inline fn jayHeight(self: FaceMetrics) f64 {
+        if (self.jay_height) |value| if (value > 0) return value;
+        return self.ascent - self.descent;
```

Comment:
Might wanna change this estimator to something more evidence based. `ascent - descent` is very likely to be too large. I have some code that measures metrics from an actual corpus of fonts, which I verified the other multiplier-based estimations with to be more or less correct for a broad range of programming fonts. I'll go ahead and figure out the best multiplier to get the maximum vertical extent of ASCII from known quantities.